core/modules/comment/comment.module

```diff
@@ -175,26 +175,16 @@ function comment_theme() {
  * Implements hook_menu().
  */
 function comment_menu() {
-  $items['admin/content/comment'] = array(
-    'title' => 'Manage comments',
-    'description' => 'List and edit site comments and the comment approval queue.',
-    'page callback' => 'comment_admin',
-    'access arguments' => array('administer comments'),
-    'type' => MENU_LOCAL_TASK | MENU_NORMAL_ITEM,
-    'file' => 'comment.admin.inc',
-  );
   // Tabs begin here.
   $items['admin/content/comment/new'] = array(
     'title' => 'Published comments',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
+    'type' => MENU_NORMAL_ITEM,
     'weight' => -10,
   );
   $items['admin/content/comment/approval'] = array(
     'title' => 'Unapproved comments',
     'title callback' => 'comment_count_unpublished',
-    'page arguments' => array('approval'),
-    'access arguments' => array('administer comments'),
-    'type' => MENU_LOCAL_TASK,
+    'type' => MENU_NORMAL_ITEM,
```

Comment:
Do we still need an access callback for this menu item?

Reply:
I've removed them from hook_menu since they're in the view.